### PR TITLE
Fix docs info bugs

### PR DESCRIPTION
**Pull request by icecrime**

### CODE REVIEW

**Comment by thaJeztah** [comment]
I see "server version 1.9" should this be in the v1.21 docs?

File: docs/reference/api/docker_remote_api_v1.22.md [hunk 1]
```diff
@@ -2133,6 +2133,7 @@ Display system-wide information
 
     {
         "Architecture": "x86_64",
+        "ClusterStore": "etcd://localhost:2379",
```
Comment (thaJeztah) [inline docs/reference/api/docker_remote_api_v1.22.md]: Looks like we need to update this in the 1.21 and 1.23 docs as well https://github.com/docker/docker/search?utf8=✓&q=DiscoveryBackend

**Comment by calavera** [comment]
ping @icecrime

**Comment by thaJeztah** [comment]
closing; carried in https://github.com/docker/docker/issues/21364

**Comment by icecrime** [comment]
Thanks, sorry :-(

**Comment by thaJeztah** [comment]
No worries, really!

**Pull request closed without merging**